docs/database/database-spans.md

```diff
@@ -225,6 +228,18 @@ Back ends could, for example, use the provided identifier to determine the appro
 
 When additional attributes are added that only apply to a specific DBMS, its identifier SHOULD be used as a namespace in the attribute key as for the attributes in the sections below.
 
+## Sanitization of `db.query.text`
+
+The `db.query.text` SHOULD be collected by default only if there is sanitization that excludes sensitive information.
+Sanitization SHOULD replace all literals with a placeholder value.
+Such literals include, but are not limited to, String, Numeric, Date and Time,
+Boolean, Interval, Binary, and Hexadecimal literals.
```

Comment:
nit: you can probably remove the "literals" on the "Hexadecimal" too, since it already starts the sentence saying those are type of literals (I saw you removed from the other on your latest commit)

Reply:
I'm not a native speak, so I just asked AI and it says it's ok to drop it, but keeping it there is more precise. It says  `"literals" at the end makes it explicitly clear that all the items in the list are types of literals`. That's also how I'd understand it - the "literals" is not on the "Hexadecimal", but on the whole list.